bower.json

```diff
@@ -4,7 +4,6 @@
     "ember": "1.13.11",
     "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
     "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-data": "1.13.15",
```

Comment:
I don't believe this is needed by this addon at all, but correct me if I'm wrong